ngraph/core/include/ngraph/function.hpp

```diff
@@ -120,6 +120,24 @@ namespace ngraph
             const std::vector<std::shared_ptr<Node>>& root_nodes)>;
         void set_topological_sort(topological_sort_t);
 
+        virtual bool visit_attributes(AttributeVisitor& visitor);
+
+        /// Return the function parameters
+        const ParameterVector& get_parameters() const { return m_parameters; };
+        /// Return a list of function's outputs
+        const ResultVector& get_results() const { return m_results; };
+        /// Index for parameter, or -1
+        int64_t get_parameter_index(const std::shared_ptr<op::Parameter>& parameter) const;
```

Comment:
Do we use this method?

Reply:
They are needed for searching Parameters in Function